lib/transformSync.js

```diff
@@ -14,7 +14,7 @@ exports.transform = function(source, options) {
 			generatedFile: path.replace(/\._coffee$/, '.coffee'),
 			sourceFiles: [path],
 		});
-		if (options.ignore && options.ignore(path)) {
+		if ((options.ignore && options.ignore(path)) || /\.coffee$/.test(options.ext || path)) {
```

Comment:
@gasi. Thanks Daniel. I could do it but then I would have to pre-process with babel because I want to keep streamline compatible with node 0.10 and 0.12. It was easier to stick to ES5.